homeassistant/components/openuv/sensor.py

```diff
@@ -104,6 +105,7 @@ class OpenUvSensorEntityDescription(
         key=TYPE_CURRENT_UV_LEVEL,
         translation_key="current_uv_level",
         icon="mdi:weather-sunny",
+        device_class=SensorDeviceClass.ENUM,
```

Comment:
Can we add an options property? Since we already know all possible values. This would increase the user experience